### PR TITLE
Full training program version

### PR DESCRIPTION
**Pull request by duchoang**
- [x] Extract code from exercise-mlp.ipynb and put it the whole python program
- [x] Add one script `run_training.sh` to run the training program
- [x] Adjust the format csv to the latest one
- [x] Fix bug loading dataset with folder
- [x] Ready for review

### CODE REVIEW

**Comment by tmbo** [comment]
:+1:

**Comment by janm399** [comment]
Overall good, but the saved file names should follow convention outlined in https://github.com/muvr/muvr-ios/pull/73.

I don't mind the exact naming convention, but it must be possible to (easily) load the model given its identifier.

**Comment by janm399** [comment]
I'm probably not understanding the code properly, but is it preparing model for each ``user_id``? I thought the intention is to have trained models for different muscle groups; for example arms, chest, back, etc.

**Comment by tmbo** [comment]
The code doesn't automatically train for multiple users, it trains a model for the passed dataset (the code doesn't know anything about users / muscle groups). Therefore, when run once the code produces one model for the passed data. It merely duplicates the notebook functionality into python scripts.

File: mlp/start_analysis.py [hunk 1]
```diff
@@ -35,7 +37,8 @@ def main(sc):
     rdd = sc.textFile(user_rdds)
     models = rdd.map(lambda user_id:
                      learn_model_from_data(dataset_directory=os.path.join(root_directory, 'datasets', user_id),
```
Comment (janm399) [inline mlp/start_analysis.py]: This is the ``user_id`` thing that threw me.

Comment (tmbo) [inline mlp/start_analysis.py]: Ok yes the spark code does train one model per user. But it needs the scala code that reads the data out of cassandra, stores it into one directory per user and hands over the directories to python.

**Pull request merged**